dv/uvm/icache/dv/ibex_icache_sim_cfg.hjson

```diff
@@ -36,6 +36,15 @@
     }
   ]
 
+  overrides: [
+    // Override the search paths to prioritize the selection of
+    // packages (with the same VLNV). The latter one wins.
```

Comment:
Nit: Maybe "The last one wins." ? Otherwise it sounds like we're only ever talking about pairs.